File: Python/Lista_02(Estrutura_Condicional)/Troco_Verificado.py
preco = float(input("Preco unitario do produto: "))

quantidade = int(input("Quantidade comprada: "))

aPagar = preco*quantidade

recebido = float(input("Dinheiro recebido: "))

if recebido>=aPagar:
    troco = recebido - aPagar
    print(f"TROCO = {troco:.2f}")
else:
    faltando = aPagar - recebido
    print(f"DINHEIRO INSUFICIENTE. FALTAM {faltando:.2f} REAIS")

File: C++/Lista_02(Estrutura_Condicional)/Troco_Verificado.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){

    double precoProduto, aPagar;
    int quantidade;
    double dinheiroRcebido, troco;

    cout << "Preco unitario do produto: ";
    cin >> precoProduto;

    cout << "Quantidade comprada: ";
    cin >> quantidade;

    aPagar = precoProduto*quantidade;

    cout << "Dinheiro recebido: ";
    cin >> dinheiroRcebido;

    if (dinheiroRcebido<aPagar){
        troco = aPagar-dinheiroRcebido;
        cout << "DINHHEIRO INSUFICIENTE. FALTAM " << fixed << setprecision(2) << troco << " REAIS.";
    }
    else{
        troco = dinheiroRcebido-aPagar;
        cout << "TROCO = " << fixed << setprecision(2) << troco;
    }

    return 0;

}